README.md

```diff
@@ -39,15 +39,16 @@ $ ez-build --help
     --interactive                  watch for and recompile on changes (implies -O 0)
     --production                   enable production options (implies -O 1)
     --flags <flags>                toggle build flags
+    @<path>                        read options from the file at <path> (relative to cwd)
 ```
 
 ## Using ez-build in npm scripts
 
 A typical scripts section of a package.json when using ez-build looks something like this:
 
 ```json
-{ "dev": "ez-build --interactive"
-, "build": "ez-build --production"
+{ "build": "ez-build --production"
```

Comment:
i would personally prefer to see `env === 'production'` or `env === 'development'` due to the nature of many other bits (like react) being aware of the `env` setting and changing output. 

then the script diffs can become `cross-env env=production ez-build`